### ISSUE TITLE
pcb

### ISSUE THREAD

**Issue by rogersung** [trash80/Arduinoboy#8]
if i dont wanna use the 面包板，how can i make the pcb (over the arduino

**Comment by trash80** [comment]
use an arduino proto-shield?

**Comment by rogersung** [comment]
so how can i buy it ~~~
i cant find it in china :(
can i contact the dealer to get the design file and make it in china

**Comment by rogersung** [comment]
okay~~~ i find in in alibaba thx sooooo much

**Comment by trash80** [comment]
yeah alibaba most likely has proto-shields if that is your question. Also seeedstudio if they sell locally.

**Comment by rogersung** [comment]
they responsed me but gave me the wrong link xD    luckily i find it in another store
thx again
